centreon/packaging/scripts/centreon-common-postinstall.sh

```diff
@@ -13,6 +13,12 @@ fixCacheConfigRights() {
   chmod 0770 /var/cache/centreon/config/broker/*
 }
 
+fixSymfonyCacheRights() {
```

Comment:
(sorry didn't see that on first review)
instead of updating `centreon-common` package, you should add this function in `centreon-web` postinstall : 
```
fixSymfonyCacheRights() {
  # MON-69138
  SYMFONY_CACHE_DIR="/var/cache/centreon/symfony"
  if [ -d "$SYMFONY_CACHE_DIR" ]; then
    if [ "$1" = "rpm" ]; then
      chown -R apache:apache "$SYMFONY_CACHE_DIR"
    else
      chown -R www-data:www-data "$SYMFONY_CACHE_DIR"
    fi
  fi
}
```
and call function with : 
```
fixSymfonyCacheRights $package_type
```